src/main/java/com/minecolonies/entity/ai/citizen/farmer/EntityAIWorkFarmer.java

```diff
@@ -537,4 +514,46 @@ public EntityCitizen getCitizen()
     {
         return worker;
     }
+
+    /**
+      * This method allows us to harvest crops and leave the plant there.
+      * Credit goes to RightClickHarvest mod.
+      *
+      * @params position the position of the crop to harvest
+      */
+    private boolean harvestCrop(final BlockPos position)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused private "harvestCrop" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUnusedPrivateMethod)